extra/mkexports

```diff
@@ -0,0 +1,27 @@
+#! /bin/sh
+# $1 - in  file
+# $2 - out file(s)
```

Comment:
It is my mistake from dabcc18ce38ca9b4a436368fc2b6c3240c4d378e: we can accept several files in $1, not write several ones ($2). Sorry for the confusion.

Reply:
I fixed this mistake)